docs/source/command_line.rst

```diff
@@ -468,7 +468,8 @@ Here are some more useful flags:
   make transformations to a file before type checking without having to change
   the file in-place.  (For example, tooling could use this to display the type
   of an expression by wrapping it with a call to reveal_type in the shadow
-  file and then parsing the output.)
+  file and then parsing the output.) This argument may be specified multiple times
+  to make mypy substitute multiple different files with their shadow replacements.
```

Comment:
This needs an example -- do you write `--shadow-file X1 X2 Y1 Y2` or `--shadow-file X1 X2 --shadow-file Y1 Y2`? (I presume the latter, but some options use the former syntax, and argparse supports both.)

Also, this argument has caused many people to be confused. We should present an example explaining that `--shadow-file X1 X2` means that whenever the checker is asked to check `X1`, it actually reads and checks the contents of `X2`, but *diagnostics will refer to `X1`* (it may well be the latter bit that trips people up).